app/controllers/events/courses_controller.rb

```diff
@@ -26,7 +26,7 @@ def index
 
   def prepare_sidebar
     @grouped_events = sorted(grouped(courses_scope, course_grouping))
-    @categories = Event::KindCategory.list
+    @categories = Event::KindCategory.list.includes(:translations)
```

Comment:
Da dieses `.list` bereits in einem Kontext definiert wird,. der translations kennt, könnten wir das doch dort auch zentral lösen, oder?

https://github.com/hitobito/hitobito/blob/28f0136212e30c85dfdd2ab01029a233e9cbddc5/app/models/concerns/globalized.rb#L36-L41

Natürlich muss man kurz prüfen, ob bei anderen `.list`-Aufrufen dann das `.includes(:translations)` wegfallen kann.